tests/features/call_proposals.feature

```diff
@@ -0,0 +1,139 @@
+    And I fill in "Media item" with "Contact image" in the "Call for proposals contact" region
+    And I fill in "Caption" with "Call for proposals contact caption" in the "Call for proposals contact" region
+    And I fill in "Press contacts" with "http://example.com/press_contacts" in the "Call for proposals contact" region
+
+    And I fill in "Alternative title" with "Alternative title 1"
+    And I fill in "Navigation title" with "Navi title 1"
+    And I fill in "Redirect link" with "http://example.com"
+
+    And I press "Save"
+    And I should see "My Call for proposals 1 body"
+    And I should see "My Call for proposals 1 reference"
+    And I should see the link "Official Journal publication 1"
+    And I should see "10/25/2020"
+    And I should see the link "http://example.com/2"
+    And I should see "Anti Fraud Information System (AFIS)"
+    And I should see "Audit Board of the European Communities"
+    And I should see "My Document 1"
+
+    And I should see the text "Name of the call for proposals contact"
+    And I should see the text "Call for proposals contact body text"
+    And I should see the text "Call for proposals contact organisation"
+    And I should see the link "http://www.example.com/call_for_proposals_contact"
+    And I should see the text "test@example.com"
+    And I should see the text "0488779033"
+    And I should see the text "0488779034"
+    And I should see the text "0488779035"
+    And I should see the text "Back street 3"
+    And I should see the text "Budapest"
+    And I should see the text "9000"
+    And I should see the text "Hungary"
+    And I should see the link "Call for proposals contact social link email"
+    And I should see the text "Call for proposals contact office"
+    And I should see the link "Contact image"
+    And I should see the text "Call for proposals contact caption"
+    And I should see the link "http://example.com/press_contacts"
+
+    And I should not see "Alternative title 1"
+    And I should not see "Navi title 1"
+
+  @javascript
+  Scenario: Constraints validation of a Call for proposals content through the UI.
+    Given I am logged in as a user with the "create oe_call_proposals content, access content, edit own oe_call_proposals content, view published skos concept entities, manage corporate content entities" permission
+
+    When I visit "the Call for proposals creation page"
+    And I fill in "Title" with "My Call for proposals 1"
+    And I set "Publication date" to the date "24-10-2020"
+    And I select "two_stage" from "oe_call_proposals_model"
+    And I fill in "Content owner" with "Committee on Agriculture and Rural Development (http://publications.europa.eu/resource/authority/corporate-body/EP_AGRI)"
+
+    And I press "Save"
+
+    Then I should see "Please select a valid date!"
+    And I set "Deadline date" to the date "31-12-2020 23:45" using format "d-m-Y H:i"
```

Comment:
we should also test with multiple values

Reply:
This is done.